app/javascript/src/Dashboard/components/NewAccountsWidget.jsx

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import React, { useState, useEffect } from 'react'
+
+import { createReactWrapper } from 'utilities'
+import { render as renderChartWidget } from 'Dashboard/chart'
+import { Spinner } from 'Common'
+import 'Dashboard/styles/dashboard.scss'
+
+type ChartValues = {
+  value: number,
+  formatted_value: string
+}
+
+type Props = {
+  chartData: {
+    values: Map<Date, ChartValues>,
+    complete: Map<Date, ChartValues>,
+    incomplete: Map<Date, ChartValues>,
+    previous: Map<Date, ChartValues>
+  },
+  newAccountsTotal: number,
+  hasHistory: boolean,
+  links: {
+    previousRangeAdminBuyersAccount: {
+      url: string,
+      value: string
+    },
+    currentRangeAdminBuyersAccount: {
+      url: string,
+      value: string
+    }
+  },
+  percentualChange: number
+}
+
+const NewAccountsWidget = ({ chartData, newAccountsTotal, hasHistory, links, percentualChange }: Props) => {
```

Comment:
I think we might be missing some typing here, right? We should specify NewAccountsWidget is a React Component, don't we?

Reply:
Right! But what the what now 😅?
What do you mean by "We should specify NewAccountsWidget is a React Component"?

Reply:
Well, there is a way to type React Components, but I see that what we usually do is [type the render](https://flow.org/en/docs/react/types/#toc-react-node). However, I don't see any Dashboard components with this typing. In any case, the way we do it is the following:

```suggestion
const NewAccountsWidget = ({ chartData, newAccountsTotal, hasHistory, links, percentualChange }: Props): React.Node => {
```

Reply:
Got it! Updated in [4eac31d](https://github.com/3scale/porta/pull/3070/commits/4eac31d5a2412d7ae81a6c241818037650e07468). Thank you!